packages/react-magma-dom/src/components/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,313 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+
+import { ThemeContext } from '../../theme/ThemeContext';
+import { ThemeInterface } from '../../theme/magma';
+import { useIsInverse } from '../../inverse';
+import { Step, StepProps, StepStatus } from './Step';
+import { transparentize } from 'polished';
+
+/**
+ * @children required
+ */
+export interface StepperProps extends React.HTMLAttributes<HTMLDivElement> {
+  testId?: string;
+  /*
+   * Custom number for specific breakpoint responsive styling
+   */
+  breakpoint?: number;
+  /*
+   * Determines the Stepper view based on a minimum breakpoint number.
+   */
+  breakpointStyle?: BreakPointStyle;
+  /*
+   * Sets a custom label for the Step count # of #.
+   */
+  stepDescriptionLabel?: string;
+  /*
+   * Sets a custom label for the completed Step.
+   */
+  completedStepDescription?: string;
```

Comment:
The name is throwing me off because it seems like it would appear after a single step is completed. Instead, this is only visible when the whole stepper is done. Let's brainstorm different name options for this, and lets update the description to make this more clear. We should also state here what the default string is (and lets check with @orion-cengage on what that default should be)

Reply:
Discussed solution: We will rename it to "completionLabel" and set the default string to "All steps completed"